lua/lspconfig.lua

```diff
@@ -76,4 +76,14 @@ function mt:__index(k)
   return configs[k]
 end
 
+local minimum_neovim_version = '0.10'
+if vim.fn.has('nvim-' .. minimum_neovim_version) == 0 then
+  local msg = string.format(
+    'Neovim version is too low. Current version: %s. Required version: %s',
```

Comment:
need to mention lspconfig, I don't think notify() does that automatically?
```suggestion
    'nvim-lspconfig requires Nvim version %s, but you are running: %s',
```